### ISSUE TITLE
Front-matter template (extra linebreak)

### ISSUE THREAD

**Issue by dxcore35** [deathau/markdownload#29]
Browser: Opera
System: Mac Os
Extension: 2.4

This is my front-matter template:

```
---

__Source:__ {baseURI}
__Author:__ {byline} 
__Language:__ English
__Type:__ Web

---

#eng

---

[[Index]] < 

---

```

This is what appears in downloaded md file:

```
---


__Source:__ Link
__Author:__  
__Language:__ English
__Type:__ Web


---


#eng



---



[[Index]] < 


---


```

The line-break is doubled for some reason.

**Comment by deathau** [comment]
I can look into this. My guess is that it will be something to do with the difference between Mac and Windows new lines

**Comment by deathau** [comment]
This turned out to be a bit different than I imagined.

I was getting the front/back matter textboxes to auto-grow by making them a contenteditble span, which for some reason on opera inserted a bunch of unnecessary divs which got translated as extra newlines when I grabbed the text.

I've changed it to be an actual textarea now, with a different mechanism for auto-resizing, and that should resolve the issue.
